lib/eco/treemanager.py

```diff
@@ -134,6 +137,8 @@ def jump_right(self):
         if self.inside() or isinstance(node, EOS):
             self.pos = len(self.node.symbol.name)
             return
+        if node.symbol.name == "\r":
+            self.line += 1
```

Comment:
Ah now I see it. I just saw the green tick in the overview and thought this means it's ready to merge which confused me.